### PR TITLE
Added logic to properly set arguments value

### PR DESCRIPTION
**Pull request by ashkan18**
# Problem
We need to properly set the arguments value when creating the query. Currently following code:
```ruby
Graphlient::Query.new do
  invoices(id: 'test') do
    name
  end
end
```

results is
```graphql
{
  invoices(id: test) {
    name
  }
}
```
Instead of `id: "test"`

# Solution
Before setting the argument check the argument class and properly set it as argument value.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
```
Comment (dblock) [inline Gemfile.lock]: Gemfile.lock should be deleted and added to .gitignore.

**Comment by dblock** [comment]
Also needs changelog.

**Comment by ashkan18** [comment]
Addressed those comments, going to merge 🙌

**Comment by dblock** [comment]
I cut 0.0.5 with this since it's a major fix.

**Pull request merged**